### PR TITLE
Full build july

### PR DESCRIPTION
**Pull request by Robinlovelace**

### CODE REVIEW

**Comment by Robinlovelace** [comment]
Heads-up @wangzhao0217 the build failed for Scotland South here:

```
✖ errored target rs_utility_quietest
✖ errored pipeline [41.331 minutes]
Warning message:
6 targets produced warnings. Run targets::tar_meta(fields = warnings, complete_only = TRUE) for the messages. 
Error:
! Error running targets::tar_make()
Error messages: targets::tar_meta(fields = error, complete_only = TRUE)
Debugging guide: https://books.ropensci.org/targets/debugging.html
How to ask for help: https://books.ropensci.org/targets/help.html
Last error message:
    Timeout was reached: [api.cyclestreets.net] Connection timeout after 10001 ms
Last error traceback:
    get_routes(od = od_utility_combined, plans = "quietest", purpose = "util...
    cyclestreets::batch(desire_lines = od, id = id, directory = folder,     ...
    batch_jobdata(username = username, password = password, id = as.characte...
    httr::POST(url = batch_url, body = body)
    request_perform(req, hu$handle$handle)
    request_fetch(req$output, req$url, handle)
    request_fetch.write_memory(req$output, req$url, handle)
    curl::curl_fetch_memory(url, handle = handle)
    .handleSimpleError(function (condition)  {     state$error <- build_mess...
    h(simpleError(msg, call))
> region_names
[1] "Glasgow and Strathclyde"   "Scotland South"           
[3] "Tayside, Central and Fife" "Edinburgh and Lothians"   
[5] "Highlands and Islands"     "Aberdeen and North East"  
```

**Comment by Robinlovelace** [comment]
This commit shows changes to restart the build at the right place: https://github.com/nptscot/npt/pull/514/commits/a744a2126ae92ce225047cfc685b2796bdb2d58a

File: parameters.json [hunk 1]
```diff
@@ -2,12 +2,12 @@
   "plans": ["fastest", "balanced", "quietest", "ebike"],
   "min_flow": [1],
   "max_to_route": [10000],
-  "date_routing": ["2024-07-01"],
```
Comment (Robinlovelace) [inline parameters.json:5]: @wangzhao0217 shouldn't the line above be 1000000?

**Review by wangzhao0217 — APPROVED** [review]

File: code/tests/data-zones-debugging.qmd [hunk 2]
```diff
@@ -24,7 +27,8 @@ file.exists(data_zones_zip_file)
 data_zones_exdir = "outputdata/DataZone_2024-07-02"
 unzip(data_zones_zip_file, exdir = data_zones_exdir)
 data_zone_files = list.files(data_zones_exdir, full.names = TRUE)
-length(data_zone_files) # 770, too few?
+length(data_zone_files) # 7457
+head(data_zone_files)
```
Comment (Robinlovelace) [inline code/tests/data-zones-debugging.qmd:31]: Can you try reproducing this code @wangzhao0217?

I get the following:

```
[1] "outputdata/DataZone_2024-07-02/c(\"S01006802\", \"S01006802\").json"
[2] "outputdata/DataZone_2024-07-02/c(\"S01006806\", \"S01006806\").json"
[3] "outputdata/DataZone_2024-07-02/c(\"S01006807\", \"S01006807\").json"
[4] "outputdata/DataZone_2024-07-02/c(\"S01006808\", \"S01006808\").json"
[5] "outputdata/DataZone_2024-07-02/c(\"S01006809\", \"S01006809\").json"
[6] "outputdata/DataZone_2024-07-02/c(\"S01007208\", \"S01007208\").json"
```

**Review by Robinlovelace — COMMENTED** [review]
Good progress but still an issue with the DataZones file names.

File: code/tests/data-zones-debugging.qmd (hunk 2 above)
Comment (Robinlovelace) [reply code/tests/data-zones-debugging.qmd:31]: Update on this...

Comment (Robinlovelace) [reply code/tests/data-zones-debugging.qmd:31]: Most of the file names are OK:

image

**Comment by Robinlovelace** [comment]
Seems to be working.

**Pull request merged**